### DISCUSSION TITLE
录音对比图中红色蓝色的点是什么？

### DISCUSSION THREAD

**Discussion by muyuxia** [ZuodaoTech/everyone-can-use-english#462] [Q&A] [answered]
![image](https://github.com/xiaolai/everyone-can-use-english/assets/11573866/72c5bf57-a623-48c1-8c1c-b1bcbef63630)

当我把鼠标悬停在红色的点的时候还会出现数字，请问点代表什么？
另外我发现我读的录音的红点高低起伏很大，有时候还会出现很多急速下降的”竖线“；而AI生成的语音上面的蓝点几乎都在中轴线以上，并且非常连贯没有太多起伏。这代表我的跟读有哪些问题呢？

谢谢您的解答。

**Comment by an-lee** [comment, marked as answer]
这个图叫 pitch contour，上面的点是音调（也就是声音频率）。跟读的时候，应该尽量模仿原音的音调变化。

目前系统对录音的音调识别可能还有较多的噪声（突然偏离太大的点），后续会继续优化。

**Reply by muyuxia** [reply]
原来如此！感谢解答：）
